int64      TRACK_ID
float64    X
float64    Y
float32    SX
float32    ANGLE
float32    W
float32    L
float32    H
float32    P1_X
float32    P1_Y
float32    P2_X
float32    P2_Y
float32    P3_X
float32    P3_Y
float32    P4_X
float32    P4_Y
float32    STATUS

